data/team2020clinical/team2020clinical-extraction.md

```diff
@@ -0,0 +1,89 @@
+# Extraction for The COVID-19 Investigation Team (2020)
+
+[The COVID-19 Investigation Team (2020)](https://www.nature.com/articles/s41591-020-0877-5) studied clinical and virologic characteristics of the first 12 patients with coronavirus disease 2019 (COVID-19) in the United States. Respiratory, stool, serum, and urine specimens were submitted for SARS-CoV-2 real-time reverse-transcription polymerase chain reaction (rRT-PCR) testing, viral culture, and whole genome sequencing. Only nasopharyngeal samples were included in this dataset. Data for the nasopharyngeal swab results were obtained from the combined dataset in the supplementary materials of [Challenger et al. (2022)](https://doi.org/10.1186/s12916-021-02220-0). Attributes of hospitalized patients were obtained from the original paper. 
+
+First, we `import` python modules needed:
+
+```python
+#import modules;
+import yaml
+import pandas as pd
+import numpy as np
+from shedding_hub import folded_str
+```
+
+Raw data, which is stored on [Shedding Hub](https://github.com/shedding-hub/shedding-hub/tree/main/data/team2020clinical), will be loaded and cleaned to match the most updated [schema](https://github.com/shedding-hub/shedding-hub/blob/main/data/.schema.yaml). 
+
+```python
+#load the data;
+df = pd.read_excel('CombinedDataset.xlsx', sheet_name='Viral_Load')
+team2020 = df[df['StudyNum'] == 7].copy()
+columns_to_drop = ['Estimated', 'SevMax', 'Sev1st', 'Died', 'SevMax3', 'Age', 'Sex']
+team2020 = team2020.drop(columns=columns_to_drop)
+
+# Information from the original paper.
+patient_info = {
+    'PatientID': ['7-6', '7-7', '7-8', '7-9', '7-10', '7-11', '7-12'],
+    'Age': [35, 65, 65, 35, 55, 55, 55],
```

Comment:
Till and I were discussing about how to format `age` information in Haisu's PR #62. As Challenger paper used average age in his dataset, we may want to include the age range instead. So how about changing the age to the range in Table 2 of Team paper (https://www.nature.com/articles/s41591-020-0877-5/tables/2) and including this information as `age_group` instead of `age`. Please see the communication for this between Till and me attached.
![image](https://github.com/user-attachments/assets/fd79a8b0-0e8b-4170-be86-bfc7cf8eed8a)

Reply:
I see. Do I need to add age_group to the schema under attribute properties, or can I still pass the check without modifying the schema?

Reply:
You can still pass the tests without having the age group in the schema. We can add it later if we see age groups cropping up across many future datasets.

Reply:
I see! Thanks!